consensus/obcpbft/obc-sieve.go

```diff
@@ -714,6 +718,7 @@ func (op *obcSieve) commit() {
 func (op *obcSieve) restoreBlockNumber() {
 	var err error
 	op.blockNumber, err = op.stack.GetBlockchainSize()
+	op.blockNumber-- // The highest block number is one less than the size
```

Comment:
Is this correct?  I changed sieve to just carry the blockchainsize as blockNumber.